regression/cbmc-incr-smt2/CMakeLists.txt

```diff
@@ -1,20 +1,24 @@
-
+# On Windows, run only z3 tests
```

Comment:
:pick: This looks a whole lot more fiddly than just installing cvc5 on the Windows job now that the pre-built binaries are available.

Reply:
Correct - now that https://github.com/diffblue/cbmc/pull/6778 has been merged, this commit is going to be purged.

Reply:
:+1: